Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(CharacterController))]
+public class PlayerMovement : MonoBehaviour
```

Comment:
This will have to be refactored in the future, that is alright, when things get more complex we might need to abstract a lot of the types we currently have.